src/chartdata/__test__/ChartData-aggregations-test.js

```diff
@@ -355,3 +406,262 @@ test('ChartData.median should return null when provided a column that doesnt exi
+});
+
+test('ChartData.quantile should memoize per column', tt => {
+    const data = new ChartData(rows, columns);
+    data.quantile('supply', 0.5);
+    tt.is(data.quantile('demand', 0.5), 56);
+});
+
+test('ChartData.quantile should return null when provided a column that doesnt exist', tt => {
+    const data = new ChartData(rows, columns);
+    tt.is(data.quantile('not here', 0.5), null);
+});
+
+test('ChartData.quantile with a p value of 0.25 should return the lower quartile', tt => {
+    const data = new ChartData(rows, columns);
+    tt.is(data.quantile('demand', 0.25), 55);
+});
+
+
+// summary
+
+test('ChartData.summary should return a five figure summary of values of a column', tt => {
+    const data = new ChartData(rows, columns);
+    const summary = data.summary('demand');
+    tt.is(typeof summary, 'object');
+    tt.is(summary.get('min'), data.min('demand'));
+    tt.is(summary.get('lowerQuartile'), data.quantile('demand', 0.25));
+    tt.is(summary.get('median'), data.median('demand'));
+    tt.is(summary.get('upperQuartile'), data.quantile('demand', 0.75));
+    tt.is(summary.get('max'), data.max('demand'));
+});
+
+test('ChartData.median should return the five figure summary of values for multiple columns', tt => {
+    const data = new ChartData(rows, columns);
+    const summary = data.summary(['supply', 'demand']);
+    tt.is(typeof summary, 'object');
+    tt.is(summary.get('min'), data.min(['supply', 'demand']));
+    tt.is(summary.get('lowerQuartile'), data.quantile(['supply', 'demand'], 0.25));
+    tt.is(summary.get('median'), data.median(['supply', 'demand']));
+    tt.is(summary.get('upperQuartile'), data.quantile(['supply', 'demand'], 0.75));
+    tt.is(summary.get('max'), data.max(['supply', 'demand']));
+});
+
+test('ChartData.summary should return a five figure summary of values of a column, even with null values', tt => {
+    const data = new ChartData(rowsWithNulls, columns);
+    const summary = data.summary('demand');
+    tt.is(typeof summary, 'object');
+    tt.is(summary.get('min'), data.min('demand'));
+    tt.is(summary.get('lowerQuartile'), data.quantile('demand', 0.25));
+    tt.is(summary.get('median'), data.median('demand'));
+    tt.is(summary.get('upperQuartile'), data.quantile('demand', 0.75));
+    tt.is(summary.get('max'), data.max('demand'));
+});
+
+test('ChartData.summary should return map with nulls when there are no values to compare', tt => {
+    const data = new ChartData(allNulls, columns);
+    tt.true(Immutable.is(data.summary('demand'), Map({
+        min: null,
+        lowerQuartile: null,
+        median: null,
+        upperQuartile: null,
+        max: null
+    })));
+});
+
+test('ChartData.summary should memoize per column', tt => {
+    const data = new ChartData(rows, columns);
+    tt.false(Immutable.is(data.summary('supply'), data.summary('demand')));
+});
+
+test('ChartData.summary should return map with nulls when provided a column that doesnt exist', tt => {
```

Comment:
Is this meant to return a map with nulls or just null? Test and descriptions don't match here